### ISSUE TITLE
make `oc` autheticated by default

### ISSUE THREAD

**Issue by kadel** [projectatomic/adb-atomic-developer-bundle#267]
It would be nice to have `oc` command authenticated by default, so user doesn't have to do `oc login` after starting the box.

**Comment by navidshaikh** [comment]
Finding and displaying the configured user(s) information for OpenShift is a feature request for vagrant-service-manager plugin as part of displaying (extra) connection information for OpenShift.

@kadel : 
Are we looking here to cut an extra step of `oc login` or trying to ease finding user and its credentials here?

**Comment by LalatenduMohanty** [comment]
@kadel what is the work flow you are following? that would be help understand the issue better.

**Comment by praveenkumar** [comment]
@LalatenduMohanty I think currently we setup 2 different user for openshift and when we deploy openshift with Vagrantfile then it will show the credentials for those users. Developer have to manually login to a user (openshift-dev or admin) before starting project. @kadel wants that should be automated (Atleast for one user).

@navidshaikh this can be feature for service-manager and will help you with it.

**Comment by LalatenduMohanty** [comment]
@praveenkumar My question is , whether the user want to do `vagrant ssh` then run oc commands or the user is outside the guest Vagrant box and using oc binary wants to communicate to the the OpenShift running in the guest Vgarant box?

**Comment by praveenkumar** [comment]
@LalatenduMohanty I was talking about `vagrant ssh` and then run `oc` command. (It might be case that @kadel was asking about from host)

**Comment by kadel** [comment]
I'm talking about `oc` inside the Vagrant box.

Because we don't ship `oc` binary for all platforms, workflow for OpenShift user looks like this:
`vagrant up`
`vagrant ssh`
`oc login`
`oc create`.... 

I think we can cut `oc login` and make it authenticated out of the box as normal user (openshift-dev).
You would run `oc login`  only when you need login as admin user.

**Comment by navidshaikh** [comment]
> @navidshaikh this can be feature for service-manager and will help you with it.

Here is the relative issue  https://github.com/projectatomic/vagrant-service-manager/issues/56

**Comment by bexelbie** [comment]
Is this also relevant? https://github.com/projectatomic/vagrant-service-manager/issues/57

**Comment by kadel** [comment]
> Is this also relevant? projectatomic/vagrant-service-manager#57

@bexelbie Not sure. My understanding of https://github.com/projectatomic/vagrant-service-manager/issues/57 is that it is about `oc` that is outside of vagrant box on host machine. In this issue I  was talking about `oc` inside vagrant box.

**Comment by praveenkumar** [comment]
@bexelbie that was bug for oc inside the vagrant box as @kadel said and it is fixed with latest PR sent on adb-utils side.
